channels/pe.m3u

```diff
@@ -51,3 +51,15 @@ https://cdnh8.iblups.com/hls/RMuwrdk7M9.m3u8
 http://170.79.102.223/UCI/index.m3u8
 #EXTINF:-1 tvg-id="VIATelevisión.pe" tvg-name="VIA Televisión" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/viatelevisionperu/picture?width=320&height=320" group-title="Local",VIA Televisión (720p)
 https://5ee0faac3bbae.streamlock.net:443/viatv2021/viatv2021/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Willax" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/willaxtv/picture?width=320&height=320" group-title="",Willax
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/WillaxTV
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/antaresTvRadioOficial/picture?width=320&height=320" group-title="Local",Antares Televisión
+https://5c3fb01839654.streamlock.net:554/iptvantares/liveantarestv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CongresoPeru/picture?width=320&height=320" group-title="",Congreso de la República (Peru)
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/channel/UCsKiP5cZCYh9YhPGrI6GrkQ
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CadenaTVHuancayo/picture?width=320&height=320" group-title="Local",Cadena TV Huancayo
+https://edge1o.live.opencaster.com/caster/kAaEdjbBiibk.m3u8
```

Comment:
Can't open that link

Reply:
Ok, I will check the link

Reply:
I couldn't get either of them to work.

Reply:
The website is https://www.cadenatv.com.pe/en-vivo/, but the broadcasting works in this web. Any suggestions?

Reply:
Casual problem: they want proper html refferer
Referrer Policy: strict-origin-when-cross-origin

Reply:
@LaneSh4d0w please maintain your hosting and make sure it resets automatically

Reply:
Like this?:
#EXTVLCOPT:http-referrer=https://www.opencaster.com
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CadenaTVHuancayo/picture?width=320&height=320" group-title="Local",Cadena TV Huancayo
https://edge1o.live.opencaster.com/caster/kAaEdjbBiibk.m3u8

Reply:
> @LaneSh4d0w please maintain your hosting and make sure it resets automatically
> https://github.com/iptv-org/iptv/pull/2933#discussion_r641739353

Thanks for noticing. It restarted.

Reply:
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CadenaTVHuancayo/picture?width=320&height=320" group-title="Local",Cadena TV Huancayo
#EXTVLCOPT:http-referrer=https://www.opencaster.com
https://edge1o.live.opencaster.com/caster/kAaEdjbBiibk.m3u8

Reply:
> #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CadenaTVHuancayo/picture?width=320&height=320" group-title="Local",Cadena TV Huancayo
> #EXTVLCOPT:http-referrer=https://www.opencaster.com
> https://edge1o.live.opencaster.com/caster/kAaEdjbBiibk.m3u8

Done